src/pages/Mypage/components/LikesList.js

```diff
@@ -37,7 +64,7 @@ const LikesList = ({ likesArr, setLikesArr, checkItems, setCheckItems }) => {
             {checkItems.length}/{likesArr.length})
           </div>
         </div>
-        <div>선택삭제</div>
+        <div onClick={() => likesDeleteFetch()}>선택삭제</div>
```

Comment:
```suggestion
        <div onClick={likesDeleteFetch}>선택삭제</div>
```
전달하는 인자가 없으면 함수 이름을 작성하는 방법으로 적용해주세요!